lib/sqlalchemy/dialects/postgresql/types.py

```diff
@@ -55,27 +55,47 @@ class BYTEA(sqltypes.LargeBinary):
 class INET(sqltypes.TypeEngine[str]):
     __visit_name__ = "INET"
 
+    def coerce_compared_value(
```

Comment:
we could make a mixin class for this, likely just otherwise empty class with this method

Reply:
Done as well.